pipelines/smartseq2_multisample/README.md

```diff
@@ -1,12 +1,64 @@
+The Smart-seq2 Multi Sample pipeline is modified from the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is designed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files. 
+
+Full details about the Smart-seq2 pipeline can be read in the [Smart-seq2 Single Sample workflow documentation](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) in GitHub. 
+ 
+## Inputs 
+
+The [Smartseq2PlateExample.json](SmartSeq2PlateExample.json) file provides example inputs for the Smart-seq2 Multi Sample workflow. 
+
+### Sample Data Input
+The sample data inputs are identical to those specified in the [Smart-seq2 Single Sample README](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/README.md). The Smart-seq2 Multi Sample workflow processes both single- and paired-end samples; however, these samples can not be mixed in the same run. 
+For the Multi Sample workflow, fastq files must be named with the following convention:
+
+*  *"sample filename prefix"*\_1.fastq (forward reads for paired-end samples or reads for single-end samples)
+*  *"sample filename prefix"*\_2.fastq (reverse reads for paired-end samples and not applicable to single-end samples)
+
+
+### Additional Input
+The reference inputs are identical to those specified in the "Additional Reference Inputs" section of the [Smart-seq2 Single Sample README](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/README.md). 
+
+In addition to the Smart-seq2 Single Sample inputs, the Multi Sample workflow requires the inputs listed in the following table:
+
+| Input name | Input Description | Input Type |
+| --- | --- | --- |
+| file_prefix | The URL path to the fastq file folder | String | 
+| input_file_names | Array of filename prefixes; workflow will be append with \_1.fastq.gz and \_2.fastq.gz | Array of Strings |
+| bacth_id | Identifier for the batch of multiple samples | String |
+| output_loom | Boolean description for optional Loom output | Boolean |
+
+ 
+ ## Smart-seq2 Multi Sample Task Summary
+ The Smart-seq2 Multi Sample pipeline calls three tasks:
+ 
+ 1) [SmartSeq2SingleSample.wdl](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl): runs the Smart-seq2 Single Sample workflow
+ 2) [SmartSeq2PlateAggregation.wdl](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/SmartSeq2PlateAggregation.wdl): moves all Zarr output files from the batch into a single directory
```

Comment:
This is the wrapper pipeline that checks the inputs, initiates per cell processing and aggregates the results